### ISSUE TITLE
Attributes such as offset and relative_to are not functional

### ISSUE THREAD

**Issue by rgerkin** [ni/nidaqmx-python#87]
Using `offset` and `relative_to` does not work on my NI-9476, despite the documentation [here](https://documentation.help/NI-Supported-Properties/NI%209476.html) saying they are supported on this device.  For example:

```python
# Numpy should be installed
import numpy as np
# Using the current version of the NI-supported nidaqmx library
from nidaqmx.constants import LineGrouping, AcquisitionType
from nidaqmx.stream_writers import DigitalSingleChannelWriter
with nidaqmx.Task() as task: # Create a new task
    # Add my NI-9476 digital output device to the task
    task.do_channels.add_do_chan("cDAQ1Mod1/port0",
                                 line_grouping=LineGrouping.CHAN_FOR_ALL_LINES)
    # Sampling rate (samples per second)
    samples_per_s = 1000
    # Length of expected data (in seconds)
    s_per_frame = 3 # Length of expected data (in samples)
    samples_per_frame = samples_per_s * s_per_frame
    # Set up sampling rate and create a buffer long enough to support multiple writes
    task.timing.cfg_samp_clk_timing(samples_per_s,
                                    sample_mode=AcquisitionType.FINITE, 
                                    samps_per_chan=samples_per_frame*5)
    # Create a channel writer for this task
    writer = DigitalSingleChannelWriter(task.out_stream,
                                        auto_start=True)
    # Write some dummy samples first (oddly, this is required to get
    # subsequent writes to work as expected)
    data = np.array([0]*2).astype(np.uint32)
    samples_written = writer.write_many_sample_port_uint32(data)
    # Report on the number of samples written, the current write position of the task,
    # and the current offset (should be 0)
    print(samples_written, task.out_stream.curr_write_pos, task.out_stream.offset)
    # Write 'samples_per_frame' worth of samples (in this case, the value 16,
    # which turns on the fourth channel (number 3 counting from 0))
    data = np.array([16]*samples_per_frame).astype(np.uint32)
    samples_written = writer.write_many_sample_port_uint32(data)
    # Report on the number of samples written, the current write position of the task,
    # and the current offset (should be 0)
    print(samples_written, task.out_stream.curr_write_pos, task.out_stream.offset)
    # Change the offset so that it will go back by 'samples_per_frame'.
    # This should bring us close to the beginning of the previous write
    task.out_stream.offset = -samples_per_frame
    # Write 'samples_per_frame' worth of samples (in this case, the value 32,
    # which turns on the fifth channel (number 4 counting from 0))
    data = np.array([32]*samples_per_frame).astype(np.uint32)
    samples_written = writer.write_many_sample_port_uint32(data)
    # Report on the number of samples written, the current write position of the task,
    # and the current offset (should be -samples_per_frame)
    # The current write position should depend on the offset, but it doesn't.  
    # Also, if you look at the behavior of the channels (by connecting
    # a device with LEDs for example),
    # You will see that the device has behaved as if the offset was 0.  
    # In other words, the values of '32' follow the values of '16'
    # rather than overwriting them, as would be implied by the offset
    print(samples_per_written, task.out_stream.curr_write_pos, task.out_stream.offset)
```

Similarly, setting `relative_to` does not work (even after applying the bug fix in #86).  Honestly, maybe a 1/3 of the API functions appear to be non-functional.

**Comment by zhindes** [comment]
This is a general NI-DAQmx problem and not an issue with the `nidaqmx` Python bindings. Consider posting on https://forums.ni.com/ for help resolving issues such as this.
